src/cljs/athens/views/athena.cljs

```diff
@@ -274,76 +270,68 @@
-                                                                                               (navigate-uid uid)))
-                                                                                 :class    (when (= i index) "selected")})
-
-                                                   [:div (use-style result-body-style)
-                                                    [:h4.title (use-sub-style result-style :title)
-                                                     [:b "Create Page: "]
-                                                     query]]
-                                                   [:span.link-leader (use-sub-style result-style :link-leader) [(r/adapt-react-class Create)]]]
-
-                                                  [:div (use-style result-style {:key      i
-                                                                                 :on-click (fn [e]
-                                                                                             (let [selected-page {:node/title   title
-                                                                                                                  :block/uid    uid
-                                                                                                                  :block/string string
-                                                                                                                  :query        query}]
-                                                                                               (dispatch [:athena/toggle])
-                                                                                               (dispatch [:athena/update-recent-items selected-page])
-                                                                                               (navigate-uid uid e)))
-                                                                                 :class    (when (= i index) "selected")})
-                                                   [:div (use-style result-body-style)
-
-                                                    [:h4.title (use-sub-style result-style :title) (highlight-match query title)]
-                                                    (when string
-                                                      [:span.preview (use-sub-style result-style :preview) (highlight-match query string)])]
-                                                   [:span.link-leader (use-sub-style result-style :link-leader) [(r/adapt-react-class ArrowForward)]]])))]))]])))})))
+  (let [s (r/atom {:index   0
+                   :query   nil
+                   :results []})]
+    [:> Dialog
+     {:fullWidth          true
+      :maxWidth           "md"
+      :onClose            #(dispatch [:athena/toggle])
+      :open               @(subscribe [:athena/open])
+      :transitionDuration {:enter 0}}
+     [:div.athena (use-style container-style)
```

Comment:
I think we need to respecify the text color here, because the MUI dialog sets a text color and isn't aware of the current Athens theme.

<img width="907" alt="CleanShot 2021-06-12 at 09 21 11@2x" src="https://user-images.githubusercontent.com/98312/121777422-f5bbf680-cb5f-11eb-88d7-0e836a442a64.png">